src/main/resources/application.conf

```diff
@@ -0,0 +1,19 @@
+sqs2kinesis {
+  sqs-endpoint = "http://0.0.0.0:4576"
+  sqs-endpoint = ${?SQS_ENDPOINT}
+  sqs-queue    = "http://0.0.0.0:4576/queue/good-events-queue"
```

Comment:
Can we append the sqs-endpoint > sqs-queue without needing to double define the value here?

Reply:
Thanks for that, I used it when testing locally and just left it by mistake. The `-endpoint` vars aren't used at the moment at all.